docs/content/latest/develop/graphql/hasura.md

```diff
@@ -37,26 +39,39 @@ For a local Mac setup, the configuration should be:
 
 ```sh
 docker run -d -p 8080:8080 \
-       -e HASURA_GRAPHQL_DATABASE_URL=postgres://postgres:@host.docker.internal:5433/postgres \
+       -e HASURA_GRAPHQL_DATABASE_URL=postgres://postgres:@host.docker.internal:5433/yugabyte \
        -e HASURA_GRAPHQL_ENABLE_CONSOLE=true \
-       hasura/graphql-engine:v1.0.0
+       hasura/graphql-engine:v1.1.0
 ```
 
 {{< note title="Note" >}}
 
-Make sure that the release version specified for `hasura/graphql-engine` matches the version you are using. The releases can be found at [Hasura graphql-engine releases](https://github.com/hasura/graphql-engine/releases).
+For the `hasura/graphql-engine` value, make sure that the version matches the Hasura GraphQL Engine version that you are using.
 
 {{< /note >}}
 
-To start Hasura, run the following script:
+After downloading or editing the `docker-run.sh` file, you need to make the file executable. Run the following `chmod` command:
 
 ```sh
-./docker-run.sh
+$ chmod +x docker-run.sh
+```
+
+When this script runs, the Hasura GraphQL Engine starts in the Docker container and returns the unique container ID.
+
+```
+$ hasura-yb ./docker-run.sh
+8afee92b0e2535baf3fdb1308f78f008b3e3c950d4f560a58449b4ef7e23652a
+```
+
+To start the Hasura GraphQL Engine, run the following script:
+
+```sh
+$ ./docker-run.sh
 ```
 
 {{< note title="Note" >}}
 
-This initialization step may take a minute or more.
+The initialization step may take a moment or more.
```

Comment:
moment and minute are very different. Not sure which one is more appropriate to use here. If initialization does take minutes, then we should say `minute` here instead of `moment`.

Reply:
Yeah, I've run this a bunch of times and never had to wait a minute for more. I'll change it to a few seconds, but how long is a moment? ;-)